index.js

```diff
@@ -0,0 +1,75 @@
+import { DateTime } from './node_modules/luxon/src/luxon.js';
+import Book from './Modules/book.js';
+import Store from './Modules/store.js';
+import UI from './Modules/ui.js';
+
+// Event: Display Books
+document.addEventListener('DOMContentLoaded', UI.displayBooks);
+// Event: Add a Book
+document.querySelector('#book-form').addEventListener('submit', (e) => {
+  // Prevent actual submit
+  e.preventDefault();
+  // Get form values
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  // Validate
+  if (title === '' || author === '') {
+    UI.showAlert('Please fill in all fields', 'danger');
+  } else {
+    // Instantiate book
+    const book = new Book(title, author);
+    // Add book to UI
+    UI.addBookToList(book);
+    // Add book to store
+    Store.addBook(book);
+    // Show success message
+    UI.showAlert('Book Added', 'success');
+    // Clear fields
+    UI.clearFields();
+  }
+});
+// Event: Remove a Book
+document.querySelector('#book-list').addEventListener('click', (e) => {
+  // Remove book from UI
+  UI.deleteBook(e.target);
+  // Remove book from local storage
+  localStorage.clear();
+});
+
+const itemOne = document.getElementById('item1');
+const itemTwo = document.getElementById('item2');
+const itemThree = document.getElementById('item3');
+const bookSection = document.getElementById('book-section');
+const addSection = document.getElementById('add-section');
+const contactSection = document.getElementById('contact-section');
```

Comment:
The navigation bar is not working as it should , reason why I do not have the possibility to check that , Kindly fix the navigation bar and make sure the `addBook` and the `removeBook` functions are working 👍